docs/website/docs/blog/2021-07-19-tflite-tosa.md

```diff
@@ -0,0 +1,38 @@
+## TFLite Support via TOSA
+
+IREE can now execute [TensorFlow Lite](https://www.tensorflow.org/lite)
+(TFLite) models through the use of
+[TOSA](https://developer.mlplatform.org/w/tosa/), an open standard of common
+tensor operations, and a part of [MLIR](https://mlir.llvm.org/) core. TOSA’s
+high-level representation of tensor operations provides a common front-end for
+ingesting models from different front-ends. In this case we ingest a TFLite
+flatbuffer, compile it to TOSA IR which IREE takes as an input format to compile
+to its various backends.
+
+![Compilation diagram](https://https://github.com/google/iree/tree/main/docs/website/docs/blog/2021-07-19-tflite-tosa-compilation-diagram.png?raw=true){ align=center }
+
+Executing TFLite on IREE provides an alternative execution environment to the
+TFLite runtime that can leverage IREE’s advantages, specifically multi-cpu and
+GPU architecture support using a lightweight execution environment. Having this
```

Comment:
This is good, but I have a few ideas for rephrasing.

* One major benefit to supporting importing from TFLite flatbuffers is that users _already_ have models stored in that format (both in locations like tfhub, and in existing apps). Supporting this path gives users more options.
* "Executing TFLite on IREE" is potentially confusing. We don't want to suggest writing an IREE delegate under TFLite or having IREE call into TFLite. What we actually have is
  1. a way to import TFLite FlatBuffer files into IREE's compiler
  2. an implementation of TFLite's C API as an optional set of bindings for using IREE's runtime
* "executing ... execution environment ... execution environment" in one sentence, should reword around that
* "multi-cpu and GPU architecture support" isn't quite how I would describe IREE's advantages right now (IREE multi-device support is still in the design phase, TFLite has GPU and TPU delegates). Check out the "Key features" on the website homepage for some ideas: https://google.github.io/iree/#key-features. I'd probably highlight ahead-of-time compilation with advanced model features along with low overhead and binary size (worded a bit differently).